src/ExplainTree.h

```diff
@@ -176,6 +196,12 @@ class LeafNode : public TreeNode {
     void render(ScreenBuffer& s) {
         s.write(xpos, ypos, txt);
     }
+
+    // print JSON
+    void printJSON(std::ostream& os, int pos) {
+        std::string tab(pos, '\t');
+        os << tab << "{ axiom: \"" << stringify(txt) << "\"}";
```

Comment:
Use quotes for keys

Reply:
Done the quoting - not happy with the code either. We need to refactor the front end. Note David is working on the new data-structures for provenance to make provenance parallel. I believe we need to complete this work prior to refactoring the command line interface.